README.md

```diff
@@ -100,3 +100,17 @@ Then follow the guide on [Executors](https://github.com/jina-ai/jina/blob/master
 Some Executors might require a large model. During CI/tests, it is advisable to download it as part of a fixture and store it to disk, to be re-used by the Executor.
 
 In production, it is recommended to set up your workspace, model, and class to load from disk. If the Executor is served with Docker, make sure to also map the directory, as the Docker runtime does not get persisted.
+
+## Using this repository as a package
+
+Notice that we have a `setup.py` in this repository. 
+This is **NOT recommended** practice for **external developers** of Executors. 
```

Comment:
Something along:

"Neither is this needed for distribution via Jina Hubble."

is missing in order to make a clear distinction between developing and distributing/using Executors.